Content.Server/Spreader/KudzuSystem.cs

```diff
@@ -52,7 +52,7 @@ private void OnKudzuSpread(EntityUid uid, KudzuComponent component, ref SpreadNe
             return;
         }
 
-        if (args.NeighborFreeTiles.Count == 0)
+        if (args.Neighbors.Count == 4)
         {
             RemCompDeferred<EdgeSpreaderComponent>(uid);
             return;
```

Comment:
Shouldn't this be getting added on in the airtight changed sub on spreadersystem? Might be better looking at that.

Reply:
Looking into it right now.
I haven't got all up in atmos and nodegroup stuff yet but AirtightComponent seems indicate if an entity blocks
atmos or not, and in what direction.

The OnAirtightChanged Event only appears to be raised by AirtightRotate(), and SetAirblocked().
Incidentally, SetAirblocked() is called by Door entities (ONLY door entities) when toggling which is the only thing I quickly found that should raise the OnAirtightChanged event on spreaders, except it doesn't seem to work. 
It should add an EdgeSpreader component to kudzu neighboring the toggling door except it doesn't. 
I haven't seen OnAirtightChanged get raised by walls at all. 

How the kudzu wakes up and keeps growing when attacked still eludes me, as AirtightChanged is the
only thing that seems to add Edgespreader components back and the kudzu isn't Airtight.

I think having the kudzu remove their EdgeSpreaderComponent only when surrounded by more kudzu like in my check is probably more performant than subscribing to all airtight or damage events anyways, but I'm new to the codebase and C#.

Reply:
OnAirtightChanged also appears to get raised on AirtightComponent ComponentShutdown which I assume
is raised when the component gets removed/deleted i.e. when I would remove a wall from the level.
Something about that just isn't working right.